Tchap/Modules/Contacts/DataSources/ContactsDataSource.m

```diff
@@ -386,7 +393,13 @@ - (void)onContactManagerDidUpdate:(NSNotification *)notif
 
 - (void)updateDirectTchapContacts
```

Comment:
Split this method in several method or use an intermediate class to handle Tchap contacts. Find a better way to handle room members asynchronous loading.

Reply:
I agree, I added this in my TODO list